Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1419,4 +1427,78 @@ - (void)serverCallback:(BNCServerResponse *)response {
     }
 }
 
+#pragma mark - Debugging functions
+
+- (void)bnc_addDebugGestureRecognizer {
+    [self bnc_addGesterRecognizer:@selector(bnc_connectToDebug:)];
+}
+
+- (void)bnc_addCancelDebugGestureRecognizer {
+    [self bnc_addGesterRecognizer:@selector(bnc_endDebug:)];
+}
+
+- (void)bnc_addGesterRecognizer:(SEL)action {
+    BNCLongPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:action];
+    BNCLongPress.cancelsTouchesInView = NO;
+    BNCLongPress.minimumPressDuration = BNCDebugTriggerDuration;
+    if (![BNCSystemObserver isSimulator]) {
```

Comment:
If you're making use of both conditions here (is and isn't), why check "not" simulator? Just put the simulator scenario first.